tower-http/src/compression_utils.rs

```diff
@@ -335,3 +336,37 @@ where
 }
 
 pub(crate) const SENTINEL_ERROR_CODE: i32 = -837459418;
+
+/// Level of compression data should be compressed with.
+#[non_exhaustive]
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Level {
```

Comment:
Would you mind renaming the type into CompressionLevel.
When reading the code, the type Level appears in a lot of places, and just `Level` is not very informative to the reader.
(Especially when coupled the decision of picking `quality` for the naming)

I think changing Level to CompressionLevel would help readers to better understand what this is about by just reading the signature.

Reply:
Makes sense. I will change that.